_data/sidebars/dg_dev_sidebar.yml

```
@@ -207,8 +207,10 @@ entries:
         url: /docs/dg/dev/backend-development/cronjobs/optimizing-jenkins-execution.html
       - title: Reduce Jenkins execution without P&S and data importers refactoring
         url: /docs/dg/dev/backend-development/cronjobs/reduce-jenkins-execution-costs-without-p&s-and-data-importers-refactoring.html
-      - title: Stable Workers
+      - title: Stable Worker
```

```suggestion
      - title: Stable Workers
```
why renaming?
The article is titled with `s` - https://docs.spryker.com/docs/dg/dev/backend-development/cronjobs/stable-workers.html